### PR TITLE
Adds two wizard healing spells, Basic Heal and Advanced Heal

### PR DESCRIPTION
**Pull request by Arzion5**
:cl:
add: Added "Basic Heal" to the wizard spellbook. It heals you for 10 brute and burn damage and has a 40-second cooldown, which can go as low as 20 seconds.
add: Added "Advanced Heal" to the wizard spellbook. It heals you for 30 brute and burn damage, 50 oxyloss and 10 clone, toxin and brain damage.
balance: No, you cannot have both at the same time.
/:cl:

Fully fledged healing spells for the wizard. It's fun! They increase survivability and have respectable cooldowns.

### CODE REVIEW

**Comment by ghost** [comment]
You currently have a negative Fix/Feature pull request delta of -2. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

**Comment by JohnGinnane** [comment]
Did you test this?

File: hippiestation/code/game/gamemodes/wizard/spellbook.dm [hunk 1]
```diff
@@ -163,6 +163,12 @@ F
 	spell_type = /obj/effect/proc_holder/spell/targeted/turf_teleport/blink
 	category = "Mobility"
 
+/datum/spellbook_entry/lesser_heal
+	name "Lesser Heal"
```
Comment (JohnGinnane) [inline hippiestation/code/game/gamemodes/wizard/spellbook.dm]: Missing an "=" homey

**Review by JohnGinnane — CHANGES_REQUESTED** [review]

**Comment by Arzion5** [comment]
Absolutely not. Re-trying

**Comment by KayeArray** [comment]
It's too weak

**Comment by Arzion5** [comment]
You're weak >:^)

Srsly tho I am probably going to buff the healing to about 20 when I'll get to coding it

**Comment by KayeArray** [comment]
Don't kill me, please!

**Comment by ghost** [comment]
ehh why not
if its too powerful or too weak it can be tweaked later
+1

**Comment by ghost** [comment]
Make it stronger but it applies an AoE effect.
Having yet another fleshmend tier ability isn't fun, having something that will save you inbetween fights is fine though.

**Comment by Arzion5** [comment]
I have absolutely no idea how to make it AoE

**Comment by JohnGinnane** [comment]
@McBawbaggings I think making it AoE defeats the purpose. I imagine the whole point of the lesser heal is a single target tiny heal with low CD, nothing fancy, nothing too powerful, just a basic little spell that can't really go wrong

**Comment by ghost** [comment]
Nice

**Comment by 2Dollarbill** [comment]
A little too hesitant on having this pass, Wizards have good mobility, adding healing to the mix might break him.

**Comment by ghost** [comment]
Your PR needs to be updated to reflect the current state of the master branch, please update your PR and either (re)open it, or ask a maintainer to (re)open it.

**Pull request closed without merging**